pkg/pool-weighted/contracts/smart/InvestmentPool.sol

```diff
@@ -115,11 +115,11 @@ contract InvestmentPool is BaseWeightedPool, ReentrancyGuard {
 
     // External functions
 
-    /**
-     * @dev Indicates whether swaps are enabled or not for the given pool.
+    /*
```

Comment:
```suggestion
    /**
```
Nitpicky as hell.

Reply:
That's actually semantically different (NatSpec or not). Good catch!